docs/faq.md

```diff
@@ -28,6 +28,21 @@ For detail information of each config field, please run:
 ```bash
 spicetify --help config
 ```
+## Spotify 'blank' after update
+
+### Windows
+
+1. It is common for this issue to stem from outdated Spotify and/or Spicetify, you can rule this out by checking your "version" and "with" entries inside of your `config-xpui.ini`.
```

Comment:
I'm not really sure what this step describes.

Reply:
It describes how to check your spotify and spicetify versions??